include/SemanticModel.h

```diff
@@ -110,6 +114,10 @@ class SemanticModel {
 
     // cache of simple integral types keyed by {width, signedness, 4-state, isReg}
     std::unordered_map<uint64_t, const TypeSymbol*> integralTypeCache;
+
+    // An empty declaration table to make it easier to create SemanticModels
+    // where we don't care about the full compilation process.
```

Comment:
I typically use doxygen comments for public-facing things. Private variables and functions are just normal comments. You're right though that I'm not the most consistent about it.